src/main/java/com/solr/clientwrapper/rest/DataIngectionResource.java

```diff
@@ -0,0 +1,62 @@
+package com.solr.clientwrapper.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import org.springframework.web.bind.annotation.RestController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.solr.clientwrapper.domain.dto.solr.SolrResponseDTO;
+import com.solr.clientwrapper.domain.service.DataIngectionService;
+
+import io.swagger.v3.oas.annotations.Operation;
+
+@RestController
+@RequestMapping("/ingection")
+public class DataIngectionResource {
+
+	private final Logger log = LoggerFactory.getLogger(DataIngectionResource.class);
+
+	@Autowired
+	DataIngectionService dataIngectionService;
+
+	@PostMapping("/batcharray/{collectionName}")
+	@Operation(summary = "/add-document", security = @SecurityRequirement(name = "bearerAuth"))
+	public ResponseEntity<String> batcharray(@PathVariable String collectionName, @RequestBody String data) {
+
+		log.debug("json array ingection ");
+		String objectdata = dataIngectionService.parseSolrSchemaArray(collectionName, data);
+		log.debug("controller :-" + objectdata.length());
+
+		if (!objectdata.isEmpty()) {
```

Comment:
According to my review of "dataIngectionService.parseSolrSchemaArray" it always returns a toString form of JSONArray so the will always be true and the false block is never executed